src/CanvasDownloadLinks.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+
+/**
+ * CanvasDownloadLinks ~
+*/
+export default class CanvasDownloadLinks extends Component {
+  fullImageLabel() {
+    const { canvasHeight, canvasWidth } = this.props;
+
+    return `Whole image (${canvasWidth} x ${canvasHeight}px)`;
+  }
+
+  smallImageLabel() {
+    const { canvasHeight, canvasWidth } = this.props;
+
+    return `Whole image (1000 x ${Math.floor((1000 * canvasHeight) / canvasWidth)}px)`;
+  }
+
+  /**
+   * Returns the rendered component
+  */
+  render() {
+    const {
+      canvasId,
+      canvasLabel,
+      canvasWidth,
+      fullImageURI,
+      smallImageURI,
+    } = this.props;
+    return (
+      <span key={canvasId}>
+        <Typography noWrap variant="h3">{canvasLabel}</Typography>
+        <List>
+          <ListItem disableGutters divider>
+            <Typography variant="body1">
+              <a href={fullImageURI} rel="noopener noreferrer" target="_blank">
+                {this.fullImageLabel()}
+              </a>
+            </Typography>
+          </ListItem>
+          {(canvasWidth > 1000)
+            && (
+            <ListItem disableGutters divider>
+              <Typography variant="body1">
```

Comment:
I believe `<Typography />` and `<a />` can be merged by the `Link` component?

Reply:
Ah, good call, I'll have to remember that for the future.